hasher-matcher-actioner/webapp/src/settings/ActionSettingsTab.jsx

```diff
@@ -0,0 +1,281 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+/* eslint-disable react/prop-types */
+
+import React, {useState} from 'react';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import Table from 'react-bootstrap/Table';
+import Form from 'react-bootstrap/Form';
+
+export default function ActionSettingsTab() {
+  return (
+    <>
+      <ActionPerformerSettings />
+      <ActionLabelSettings />
+    </>
+  );
+}
+
+function ActionLabelSettings() {
+  return <>Action Label Settings coming here</>;
+}
+
+const ActionerTypes = {
+  WebhookActioner: {
+    args: {
+      url: {description: 'The url to send a webhook to'},
+      webhookType: {
+        description: 'What type of webhook should be sent?',
+        default: 'POST',
+      },
+      headers: {
+        description: 'Optional json object of headers to include in webhook',
+        default: '{}',
+      },
+    },
+    description:
+      'When a match occurs, a webhook will be sent to the specified url with data describing the match',
+  },
+};
+
+function WebhookActionerRender({url, webhookType, headers}) {
+  const actionerDetails = ActionerTypes.WebhookActioner;
+
+  const [editing, setEditing] = useState(false);
+
+  const [savedURL, setURL] = useState(url);
+  const [savedWebhookType, setWebhookType] = useState(webhookType);
+  const [savedHeaders, setHeaders] = useState(headers);
+  const [tempURL, setTempURL] = useState(url);
+  const [tempWebhookType, setTempWebhookType] = useState(webhookType);
+  const [tempHeaders, setTempHeaders] = useState(headers);
+
+  return (
+    <Card>
+      <div hidden={editing}>
+        <Card.Header>
+          WebhookActioner
+          <br />
+          <Form.Text className="text-muted">
+            {ActionerTypes.WebhookActioner.description}
+          </Form.Text>
+        </Card.Header>
+        <Card.Body>
+          URL : {savedURL}
```

Comment:
Still looking for formatting some other way (use a table, for example). Not required for this PR.